test/schemas/test/changelogs/maximal/changelog.yaml

```diff
@@ -32,6 +32,8 @@ releases:
         - something that is not included in release notes
       known_issues:
         - som other
+      xxx:
+        - we should ignore unknown keys because used can define custom section in changelogs/config.yaml file
```

Comment:
we should ignore unknown keys because `user` can define custom section in changelogs/config.yaml file